### DISCUSSION TITLE
Difference between mal and mdb_myanimelist

### DISCUSSION THREAD

**Discussion by alexkahler** [meisnate12/Plex-Meta-Manager#1897] [Q&A] [answered]
What's the difference between mal and mdb_myanimelist as values in the mass_***_rating_update? If I want the average rating as listed on MAL, which should I choose?

**Comment by YozoraXCII** [comment, marked as answer]
`mal` fetches the information straight from MyAnimeList, whilst `mdb_myanimelist` grabs it from mdblist - the values may be the same, but mdblist tends to be slightly more out of date. We present all options, even if some are duplicated across services.
